packages/xo-web/src/icons.scss

```diff
@@ -583,6 +583,14 @@
   // SR
   &-sr,
   &-vdi {
+    &-disable {
+      @extend .fa;
+      @extend .fa-times-circle;
+    }
+    &-enable {
+      @extend .fa;
+      @extend .fa-check-circle;
```

Comment:
What about [`.fa-wrench`](https://fontawesome.com/v4/icon/wrench)?

Reply:
It was to use the same icon than [host maintenance button](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/icons.scss#L638), but i agree that `.fa-wrench` is more appropriate.